vpn_manager.sh

```diff
@@ -121,6 +121,29 @@ function create_droplet() {
             "ipv6": true,
             "tags": ["'"$TAG"'"]
         }')
+    
+    # Check if the API call was successful
+    if [[ $? -ne 0 ]]; then
+        echo "Error: Failed to create droplet."
+        exit 1
+    fi
```

Comment:
This is duplicate of L148 - 151

Reply:
Corrected the code and removed the duplicates